pages/pools/[pool].tsx

```diff
@@ -199,33 +199,39 @@ export default function Pool() {
             />
             <>
               <StyledDivForCards>
-                <ManageLiquidityCard
-                  providedLiquidityReserve={pool.liquidity.reserves.provided}
-                  providedLiquidity={pool.liquidity.available.provided}
-                  stakedLiquidityReserve={
-                    pool.liquidity.reserves.providedStaked
-                  }
-                  providedTotalLiquidity={pool.liquidity.providedTotal}
-                  stakedLiquidity={pool.liquidity.staked}
-                  tokenASymbol={tokenA.symbol}
-                  tokenBSymbol={tokenB.symbol}
-                  supportsIncentives={supportsIncentives}
-                  onClick={() =>
-                    setManageLiquidityDialogState({
-                      isShowing: true,
-                      actionType: 'add',
-                    })
-                  }
-                />
-                <ManageBondedLiquidityCard
-                  onClick={() => setIsBondingDialogShowing(true)}
-                  providedLiquidity={pool.liquidity.available.provided}
-                  stakedLiquidity={pool.liquidity.staked.provided}
-                  yieldPercentageReturn={
-                    pool.liquidity.rewards.annualYieldPercentageReturn
-                  }
-                  supportsIncentives={supportsIncentives}
-                />
+                <div style={{ flexBasis: 'auto', flexGrow: 1 }}>
+                  <ManageLiquidityCard
+                    providedLiquidityReserve={pool.liquidity.reserves.provided}
+                    providedLiquidity={pool.liquidity.available.provided}
+                    stakedLiquidityReserve={
+                      pool.liquidity.reserves.providedStaked
+                    }
+                    providedTotalLiquidity={pool.liquidity.providedTotal}
+                    stakedLiquidity={pool.liquidity.staked}
+                    tokenASymbol={tokenA.symbol}
+                    tokenBSymbol={tokenB.symbol}
+                    supportsIncentives={supportsIncentives}
+                    onClick={() =>
+                      setManageLiquidityDialogState({
+                        isShowing: true,
+                        actionType: 'add',
+                      })
+                    }
+                  />
+                </div>
+                <div style={{ flexBasis: 'auto', flexGrow: 1 }}>
+                  <ManageBondedLiquidityCard
+                    onClick={() => setIsBondingDialogShowing(true)}
+                    providedLiquidity={pool.liquidity.available.provided}
+                    stakedLiquidity={pool.liquidity.staked.provided}
+                    yieldPercentageReturn={
+                      pool.liquidity.rewards.annualYieldPercentageReturn
+                    }
+                    supportsIncentives={supportsIncentives}
+                  />
+                </div>
+              </StyledDivForCards>
+              <div style={{ marginTop: '12px' }}>
```

Comment:
If you use a styled component, like inline or column you could access our tokenized offsets. So 12px would be just `$6` if you were to pass the same styles in the css prop.